app/renderer/components/preferences/payment/ledgerTable.js

```diff
@@ -113,11 +113,11 @@ class LedgerTable extends ImmutableComponent {
   get columnClassNames () {
     return [
       css(styles.alignRight, styles.verifiedTd), // verified
-      css(styles.alignRight), // sites
-      css(styles.alignLeft),  // include
+      css(styles.alignRight, styles.column_sites), // sites
+      css(styles.alignCenter),  // include
```

Comment:
was this done intentionally? we are not using align center anywhere else

Reply:
It is intentional as the switches are aligned to the center as well.

Reply:
Check the case like this:

Before:
<img width="121" alt="screenshot 2017-09-11 16 57 40" src="https://user-images.githubusercontent.com/3362943/30264100-5914bf5e-9712-11e7-9264-d20c9e0c344f.png">

After:
<img width="118" alt="screenshot 2017-09-11 16 57 17" src="https://user-images.githubusercontent.com/3362943/30264101-59182e96-9712-11e7-8d6c-675826eca4dd.png">